### PR TITLE
Rename WRITE A POST to CREATE POST

### PR DESCRIPTION
**Pull request by lisasy**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Update the naming of the create post entrypoint from "Write a post" to "Create post" optimizing for a few things:
1. this makes clear that this composer isn't going to be limited to merely long-form essay format.
2. the new language is more concise

### Before
Screen Shot 2021-05-25 at 8 37 00 AM

### Proposed
Screen Shot 2021-05-25 at 8 37 19 AM

### UI accessibility concerns?
None

## Added tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests
- [x] I modified tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [x] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/views/pages/contact.html.erb [hunk 1]
```diff
@@ -39,7 +39,7 @@
       To report a bug, please create <a href="https://github.com/forem/forem/issues/new/choose">a bug report</a> in our open source repository.
     </p>
     <p>
-      To request a feature, please visit <a href="https://forem.dev">forem.dev</a> and write a post!
+      To request a feature, please visit <a href="https://forem.dev">forem.dev</a> and Create Post!
```
Comment (Zhao-Andy) [inline app/views/pages/contact.html.erb:42]: This should probably still be "write a post" 😅

**Review by Zhao-Andy — CHANGES_REQUESTED** [review]
One small change, as well as some merge conflicts, but otherwise looks good :)

File: app/views/pages/contact.html.erb (hunk 1 above)
Comment (lisasy) [reply app/views/pages/contact.html.erb:42]: @Zhao-Andy 😅 you're right (or "write"?) let me fix this!

Comment (lisasy) [inline app/views/pages/contact.html.erb:42]:
```suggestion
      To request a feature, please visit <a href="https://forem.dev">forem.dev</a> and write a post!
```

**Comment by lisasy** [comment]
Closing this PR because I messed up :( Starting over in another PR.

**Pull request closed without merging**